app/controllers/govuk_publishing_components/component_guide_controller.rb

```diff
@@ -86,24 +86,30 @@ def components_in_use
 
       files.each do |file|
         data = File.read(file)
-        matches << data.scan(/(govuk_publishing_components\/components\/[a-z_-]+)/)
+        matches << data.scan(/(govuk_publishing_components\/components\/[\/a-zA-Z_-]+['"]{1})/)
```

Comment:
Thanks. Actually I've realised I don't need that bit on the end of the regexp at all, so I've removed both bits 👍

Reply:
Cool, is that because of the check for the file existing? I guess it now expects something like thumbnail.svg to be a component but just doesn't find a thumbnail.html.erb?

Probably still worth using the look behind to avoid you needing the `m.gsub("govuk_publishing_components/components/", "")` twice in the code.

Reply:
Ah, of course. Your regex-fu is stronger than mine. Thanks, that's much better.